src/jira/client/index.ts

```diff
@@ -274,9 +274,9 @@ export default async (
  */
 const batchedBulkUpdate = async (
 	data,
-	options,
 	instance: AxiosInstance,
-	installationId: number
+	installationId: number,
+	options?: { preventTransitions: boolean }
```

Comment:
I wonder is it related to logging or you are also trying to fix passing `preventTransitions` to Jira in this PR?

Reply:
Not related to logging, was just fixing a type